recursion/compiler/src/ir/utils.rs

```diff
@@ -130,6 +130,41 @@ impl<C: Config> Builder<C> {
         result
     }
 
+    /// A version of `exp_reverse_bits_len` that uses the ExpReverseBitsLen precompile.
+    pub fn exp_reverse_bits_len_fast(
+        &mut self,
+        x: Felt<C::F>,
+        power_bits: &Array<C, Var<C::N>>,
+        bit_len: impl Into<Usize<C::N>>,
+    ) -> Felt<C::F> {
+        // Instantiate an array of length one and store the value of x.
+        let mut x_copy_arr: Array<C, Felt<C::F>> = self.dyn_array(1);
+        self.set(&mut x_copy_arr, 0, x);
```

Comment:
nit: try to put space above with lines that have "//"

Reply:
Rust analyzer gets rid of that white space because it's at the beginning of the function body.

Reply:
line below it